.github/workflows/packages.yml

```diff
@@ -28,3 +28,32 @@ jobs:
           path: |
             dist/*.tar.gz
             dist/*-manylinux1_*.whl
+
+  build-windows:
+    name: Build Windows packages
+    runs-on: windows-latest
+    strategy:
+      max-parallel: 100
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+
+    steps:
+    - name: Checkout repos
+      uses: actions/checkout@v1
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Build x86_64 packages
+      shell: cmd
+      run: |
+        pip install wheel || exit /b
```

Comment:
What is the `|| exit /b` for?

Reply:
It ensures that the job exits when encountering a non-zero exit code: http://paste.openstack.org/show/800743/

For what is worth, Windows exit code handling can be quite inconsistent, so in some cases you need to be explicit.

Reply:
I guess it would be better to define these two commands as two different commands, like with tox for linux:
```
      - name: Install Tox
        run: pip install tox

      - name: Run tests
        run: tox -e ${{matrix.python-version}} -- -vrsx --color=yes
```

Reply:
Right, this does look better. I'm updating the commit, thanks.